src/main.tsx

```diff
@@ -1,22 +1,50 @@
 import "./css/style.scss";
 
 import { render } from "preact";
-import { banner, accordion } from "identity-style-guide";
+import { banner, accordion, navigation } from "identity-style-guide";
 import { Link } from "./router";
 import { Routes } from "./routes";
 
-[banner, accordion].forEach((component) => component.on());
+[banner, accordion, navigation].forEach((component) => component.on());
 
 render(
   <div>
-    <nav>
-      <div>
-        <Link href="/">Home</Link>
+    <header className="usa-header usa-header--extended">
+      <div className="usa-navbar">
+        <div className="usa-logo">
+          <a href="/" title="Home" aria-label="Home">
```

Comment:
`<Link>` here?

Reply:
done in 40d5a9cfd38d006b2ec14ff57736c7d5688f689a